cpu/cortexm_common/thread_arch.c

```diff
@@ -317,7 +317,10 @@ void __attribute__((naked)) __attribute__((used)) isr_pendsv(void) {
 #endif
     "push   {lr}                      \n" /* push exception return code */
 
+    /* Disable interrupts during scheduling */
+    "cpsid  i                         \n"
```

Comment:
could you move this comment after the next line? or would it get too long?

Reply:
Reworded it a bit to make it fit :)